ci/travis/copyright-format.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+function usage()
+{
+  echo "Usage: copyright-format.sh [<args>]"
+  echo
+  echo "Options:"
+  echo "  -h|--help               print the help info"
+  echo "  -c|--check              check whether there are format issues in C++ files"
+  echo "  -f|--fix                fix all the format issue directly"
+  echo
+}
+
+pushd "$ROOT_DIR"/../..
+
+COPYRIGHT_FILE="$ROOT_DIR"/copyright.txt
+
+COPYRIGHT=$(cat "$COPYRIGHT_FILE")
+
+LINES_NUM=$(echo "$COPYRIGHT" | wc -l)
+
+RUN_TYPE="diff"
+
+FILE_LIST_TMP_FILE=".cr_file_list_tmp"
```

Comment:
Can we avoid temp files? If we can't, at least put them into `/tmp`, not the current directory.

Reply:
I can move to `/tmp`.